charts/node-analyzer/README.md

```diff
@@ -1,333 +1,220 @@
 # Chart: Node Analyzer
 
-The Node Analyzer (NA) provides a method for deploying the components for three different Sysdig Secure features:
+## Overview
 
-The Node Analyzer is deployed by default unless you set the value `nodeAnalyzer.deploy` to `false`.
+The Node Analyzer (NA) provides a method for deploying the components for the following Sysdig Secure features:
```

Comment:
I don't think we use NA as an abbreviation for node analyzer

Reply:
will remove